.github/workflows/test.yml

```diff
@@ -188,9 +188,9 @@ jobs:
         with:
           file: ./coverage.txt
         if: "env.GIT_DIFF != ''"
-  test-unit-proto-expected-to-fail:
+  test-unit-amino-might-fail:
```

Comment:
lol

Reply:
why the `might-fail`?

Reply:
Because when we remove `HybridCodec` in #6837, it will start failing until we finish #6190.